packages/aragon-wrapper/src/cache/index.js

```diff
@@ -9,22 +10,33 @@ export default class Cache {
   constructor (prefix) {
     this.prefix = prefix
 
-    // if (typeof window === 'undefined') {
-    //   // TODO: Support caching on the file system
-    //   const path = require('path')
-    //   const os = require('os')
-    // }
-
-    // Set up the changes observable
-    this.changes = new Subject()
-
     // Set up cache DB
     this.db = localforage.createInstance({
       driver: [localforage.INDEXEDDB, localforage.LOCALSTORAGE],
       name: `localforage/${this.prefix}`
     })
   }
 
+  async init () {
+    // Set up the changes observable
+    this.changes = new Subject()
+
+    try {
+      await this.db.ready()
```

Comment:
This was really fun to debug; localforage has a race condition when there's no storage providers available if you try to set new drivers before waiting for `localforage.ready()`'s promise to resolve.